composer.json

```diff
@@ -28,7 +28,7 @@
     }
   ],
   "require": {
-    "php": "^7.4 || ^8.0",
+    "php": "^7.4 || ^8.0, <8.2",
```

Comment:
We support PHP 7.4, 8.0 and 8.1. Don't suggest we support 8.2, which will be entering development shortly, until we've tested that.